Monika After Story/game/sprite-chart.rpy

```diff
@@ -1873,7 +1873,7 @@ image monika 1wubso = DynamicDisplayable(
     blush="shade"
 )
 
-image monika 1wubsw = DynamicDisplayable(
```

Comment:
wait, `wubsw` is one of the expressions we chose, not `wuw`.
```
w - wide eyes
u - up eyebrows
bs - blush shade
w - wide mouth
```

So the mouths should have been wide, not small.